tests/platform_tests/conftest.py

```diff
@@ -257,8 +261,8 @@ def advanceboot_loganalyzer(duthosts, rand_one_dut_hostname, request):
             pytest.skip('Testcase not supported for kvm')
 
     loganalyzer = LogAnalyzer(ansible_host=duthost, marker_prefix="test_advanced_reboot_{}".format(test_name),
-                    additional_files={'/var/log/swss/sairedis.rec': 'recording on: /var/log/swss/sairedis.rec'})
-    marker = loganalyzer.init()
+                    additional_files={'/var/log/swss/sairedis.rec': 'recording on: /var/log/swss/sairedis.rec', '/var/log/frr/bgpd.log': ''})
```

Comment:
If there is no start string specified for a log file, loganalyzer would look for a start string in a default format. The following change means to add the default start string for bgpd.log https://github.com/Azure/sonic-mgmt/blob/6847894fe8cb47151756c360407d3c4523cebe84/tests/platform_tests/conftest.py#L265
The reason not to use the same way for sairedis.rec is because sairedis.rec has a distinct format for starting and has a strict format for sai player purpose. A default start string would affect its format.